frontend/src/pages/agreements/list/AgreementsList.jsx

```diff
@@ -36,11 +36,12 @@ export const AgreementsList = () => {
         data: agreements,
         error: errorAgreement,
         isLoading: isLoadingAgreement,
-        refetch,
+        refetch, // is this needed?
```

Comment:
@johndeange Do you know if we need this?

Reply:
We have needed that previously - not sure if it still is post `opsApi` changes.